### ISSUE TITLE
1.6.1: cannot find module '../formatters/formatter'

### ISSUE THREAD

**Issue by XhmikosR** [GoogleChrome/lighthouse#1897]
So, I'm getting this after the 1.6.1 update:

```
C:\>node -v && npm -v
v6.10.1
3.10.10

C:\>lighthouse --version
module.js:471
    throw err;
    ^

Error: Cannot find module '../formatters/formatter'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\Users\xmr\AppData\Roaming\npm\node_modules\lighthouse\lighthouse-core\report\report-generator.js:21:19)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)

```

**Comment by XhmikosR** [comment]
It seems the .npmignore from the published package includes changes which aren't in the repository, at least not in the master branch:

```
_myscripts/
lighthouse-core/config/a11y.json
lighthouse-core/config/perf-w-artifacts.json
lighthouse-core/formatters/
lol.json
random traces/
thing.patch
traces-w-metrics.zip

```

**Comment by thasmo** [comment]
So, `1.6.1` is currently not working at all, I guess?

**Comment by XhmikosR** [comment]
Yup, we need a new patch release ASAP.

**Comment by GabLeRoux** [comment]
Failed for me too. Until its fixed:
```bash
npm uninstall -g lighthouse
npm install -g lighthouse@1.6.0
```

**Comment by paulirish** [comment]
Thanks for reporting! I published 1.6.3 which fixes the error.
